### PR TITLE
Add style.css, scripts.js and index.html

### PR DESCRIPTION
**Pull request by jbej**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
```
Comment (matwjablonski) [inline index.html]: add attribute lang

File: index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<head>
+    <meta charset="utf-8">
+    <title>Kanban board</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
```
Comment (matwjablonski) [inline index.html]: unnecessary empty line - to remove

File: index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	<div id="board" class="board">
+        <h1>Kanban board</h1>
+        <button class="create-column">Add a column</button>
+        <div class="column-container"></div>
+	</div>
+
```
Comment (matwjablonski) [inline index.html]: unnecessary empty line - to remove

File: scripts.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+            var $columnTitle = $('<h2>').addClass('column-title').text(self.name);
+            var $columnCardList = $('<ul>').addClass('column-card-list');
+            var $columnDelete = $('<button>').addClass('btn-delete').text('x');
+            var $columnAddCard = $('<button>').addClass('add-card').text('Add a card');
+        
+            $columnDelete.click(function() {
+                self.removeColumn();
+            });
+        
+            $columnAddCard.click(function() {
```
Comment (matwjablonski) [inline scripts.js]: better use method `on` instead of `click` - in whole project

**Review by matwjablonski — CHANGES_REQUESTED** [review]
can you move card to empty column?

**Comment by jbej** [comment]
Yes, I can move cards.

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**